scripts/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM erlang:latest
```

Comment:
I think it would be good to pin this to a specific version as you had mentioned in the comments. `22.2.3` should work.

Reply:
Yup, done 👍